tests/redirects_file_test.go

```diff
@@ -232,38 +150,19 @@ func TestRedirectsFileSupport(t *testing.T) {
 			//	test_should_not_contain "my 404" response
 			//
 			// '
-			// {
-			// 	// TODO: confirm with lidel: this should be skipped
-			// 	Name: "This test ensures _redirects is supported only on Web Gateways that use Host header (DNSLink, Subdomain)",
-			// 	Hint: `
-			// We expect the request to fail with a 404 (do not use the _redirect), and that 404 should not contain the custom 404 body.
-			// `,
-			// 	Request: Request().
-			// 		URL("http://127.0.0.1:8080/ipfs/{{etag}}/301-redirect-one", redirectDirCID),
-			// 	Response: Expect().
-			// 		Status(404).
-			// 		Body(Not(Contains(custom404.ReadFile()))),
-			// },
+			// TODO(lidel): Do we want to explicitly test something like "if the spec `redirect` is not enabled, then test this case"?
+			// 				Recommendation: don't (because you might disable a spec just because it's not fully implemented yet, which is different from not supporting it)
+			// 								and keep the test in kubo sharness.
```

Comment:
@lidel @hacdias highlight over this test (see the bash script comments above the diff)